source/installation-guide/compatibility_matrix/index.rst

```diff
@@ -21,32 +21,33 @@ Wazuh manager
-+-----------------------------------+------------------------+--------+--------+
-|   Oracle Linux 6 or newer         |       ✓                |   ✗    |   ✗    |
-+-----------------------------------+------------------------+--------+--------+
-|   RHEL 6 or newer                 |       ✓                |   ✓    |   ✓    |
-+-----------------------------------+------------------------+--------+--------+
-|   SUSE 12* or newer               |       ✓                |   ✗    |   ✗    |
-+-----------------------------------+------------------------+--------+--------+
-|   Ubuntu Precise/12.04 - 16.10    |       ✓                |   ✓    |   ✓    |
-+-----------------------------------+------------------------+--------+--------+
-|   Ubuntu Zesty/17.04 or newer     |       ✓                |   ✗    |   ✗    |
-+-----------------------------------+------------------------+--------+--------+
-
++-----------------------------------+--------------------------------------------------------------------------+
+|                                   |   **Wazuh version**                                                      |
++    **Operating System**           +-------------------------------+------------------------+--------+--------+
+|                                   |   From 3.13.x to current      |  From 3.0.x to current |  2.1.1 |  2.1.0 |
++-----------------------------------+-------------------------------+------------------------+--------+--------+
+|   Amazon Linux 1 or newer         | ✓                             | ✓                      |   ✓    |   ✓    |
++-----------------------------------+-------------------------------+------------------------+--------+--------+
+|   CentOS 6 or newer               | ✓                             | ✓                      |   ✓    |   ✓    |
++-----------------------------------+-------------------------------+------------------------+--------+--------+
+|   Debian Wheezy/7 or newer        | ✓                             | ✓                      |   ✓    |   ✓    |
++-----------------------------------+-------------------------------+------------------------+--------+--------+
+|   Fedora (>= 22)                  | ✓                             | ✓                      |   ✓    |   ✓    |
++-----------------------------------+-------------------------------+------------------------+--------+--------+
+|   OpenSUSE 42 or newer            | ✓                             | ✓                      |   ✗    |   ✗    |
++-----------------------------------+-------------------------------+------------------------+--------+--------+
+|   Oracle Linux 6 or newer         | ✓                             | ✓                      |   ✗    |   ✗    |
++-----------------------------------+-------------------------------+------------------------+--------+--------+
+| Raspbian Pi OS Stretch/9 or newer | ✓                             | ✗                      |   ✗    |   ✗    |
```

Comment:
This OS is not indented. you should indent Raspbian, starting at the same point that all others, and expand the entire table.